doc/sphinx/Pacemaker_Explained/status.rst

```diff
@@ -1,375 +1,374 @@
+   |                  |                                                          |
+   |                  | Is the node a member at the cluster communication later? |
+   |                  | Allowed values: ``true``, ``false``.                     |
+   +------------------+----------------------------------------------------------+
+   | crmd             | .. index::                                               |
+   |                  |    single: crmd; Node Status                             |
+   |                  |    single: Node; Status, crmd                            |
+   |                  |                                                          |
+   |                  | Is the node a member at the pacemaker layer?  Allowed    |
+   |                  | values: ``online``, ``offline``.                         |
+   +------------------+----------------------------------------------------------+
+   | crm-debug-origin | .. index::                                               |
+   |                  |    single: crm-debug-origin; Node Status                 |
+   |                  |    single: Node; Status, crm-debug-origin                |
+   |                  |                                                          |
+   |                  | The name of the source function that made the most       |
+   |                  | recent change (for debugging purposes).                  |
+   +------------------+----------------------------------------------------------+
+   | join             | .. index::                                               |
+   |                  |    single: join; Node Status                             |
+   |                  |    single: Node; Status, join                            |
+   |                  |                                                          |
+   |                  | Does the node participate in hosting resources?          |
+   |                  | Allowed values: ``down``, ``pending``, ``member``.       |
+   |                  | ``banned``.                                              |
+   +------------------+----------------------------------------------------------+
+   | expected         | .. index::                                               |
+   |                  |   single: expected; Node Status                          |
+   |                  |   single: Node; Status, expected                         |
+   |                  |                                                          |
+   |                  | Expected value for ``join``.                             |
+   +------------------+----------------------------------------------------------+
+   
+The cluster uses these fields to determine whether, at the node level, the
+node is healthy or is in a failed state and needs to be fenced.
+   
+Transient Node Attributes
+#########################
+   
+Like regular `Node Attributes`_, the name/value
```

Comment:
Obviously we can't fix this until we convert the nodes chapter, but to refer to another section, use

    :ref:`unique_id`

where this is just before the point being referred to:

    .. _unique_id:

If the thing being referred to is a section title, that will be substituted for the reference.